3rdparty/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
-if (ECM_FOUND)
-    include(ECMEnableSanitizers)
-endif()
+if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    if (ECM_FOUND)
```

Comment:
the ECM_FOUND conditional should be outside the branch, such that we can still build with sanitizers on non-linux platforms

Reply:
For some reason that didn't work with the extra-cmake-modules coming from homebrew. I'll investigate what is the problem there. I agree sanitizers should work for macOS as well.